public/javascript/logout.js

```diff
@@ -0,0 +1,9 @@
+const logoutBtn = document.querySelector('#logout');
+
+const logout = () => {
+  localStorage.clear();
```

Comment:
Shouldn't you remove just the token?

Reply:
Ok. I will do just that.
Thanks for the feedback.